examples/vector_return/Cargo.lock

```diff
@@ -0,0 +1,14 @@
+[root]
+name = "vector_return"
+version = "0.1.0"
+dependencies = [
+ "libc 0.2.16 (registry+https://github.com/rust-lang/crates.io-index)",
+]
+
+[[package]]
+name = "libc"
+version = "0.2.16"
```

Comment:
I'm pulling the rug out from under you a little bit by doing my own updates to 0.2.17 (#44). It'd be super great if you could match.